ambari-server/src/main/java/org/apache/ambari/server/controller/internal/StackServiceComponentResourceProvider.java

```diff
@@ -62,6 +62,9 @@
   private static final String COMPONENT_CATEGORY_PROPERTY_ID = PropertyHelper.getPropertyId(
       "StackServiceComponents", "component_category");
 
+  private static final String COMPONENT_VERSION_PROPERTY_ID = PropertyHelper.getPropertyId(
```

Comment:
Should also be added to `propertyIds`:

https://github.com/apache/ambari/blob/48d882d1b797cbb355a1cbc53389e2991b72ee04/ambari-server/src/main/java/org/apache/ambari/server/controller/internal/StackServiceComponentResourceProvider.java#L124-L127